### DISCUSSION TITLE
How to quick Edit Layouts

### DISCUSSION THREAD

**Discussion by Sebo848** [Novakasa/brickrail#178] [General]
Is it possible to make quick edits like:

- delete a section of track - sometimes I click and make track that I don't want would be good to have a quick edit undo function 
- delete a train - I have added more trains than I have and want to delete one 
- delete a block or edit a block move it make shorter longer. I have added blocks that I don't want anymore or I want 2 instead of 1 

Maybe these functions are there and I am just yet to find them.

**Comment by Novakasa** [comment]
In general, you can select something and press the delete key on your keyboard. There unfortunately is no button in the UI for it.
When deleting a block, the markers might not be deleted with it, make sure you delete previous markers in the section where you want to place a new block.

**Reply by Sebo848** [reply]
Hmm I thought that should work but it doesn’t… I noticed that the app icon is also blank…. Maybe I need to grab a fresh download and do some local clean up…

**Reply by Sebo848** [reply]
So I found the solution. As I am running Windows VM on Mac, in order to use the delete function and not backspace Windows requires the sequence FN+delete - all working as described!

**Reply by Novakasa** [reply]
alright, nice. Though I defininitely should've added a delete button, tracking this in #179
